.eslintrc.cjs

```diff
@@ -0,0 +1,24 @@
+'use strict'
```

Comment:
ES modules [is not supported yet](https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats) by the ESLint configuration file.